packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_interface/webview_platform.dart

```diff
@@ -59,6 +59,7 @@ abstract class WebViewPlatform {
   /// Clears all cookies for all [WebView] instances.
   ///
   /// Returns true if cookies were present before clearing, else false.
+  @Deprecated('Use `WebViewCookieManagerPlatform.clearCookies` instead.')
```

Comment:
@Reviewer Just want to double check if it's ok to deprecate platform interface methods like this?

Reply:
You can't deprecate methods we're still using; it breaks the tree. (I should have tooling soon that will catch this kind of thing in CI.)

They can be commented as deprecated, and then actually deprecated later once the uses have been removed.

Unless you can fix the call sites right away, we need to revert this part, as this will close not only our tree, but the flutter/flutter tree as well.

Reply:
@stuartmorgan Alright, I'll quickly submit a PR that changes this annotation to a comment.